scripts/cloudant/manage_cloudant.py

```diff
@@ -8,6 +8,12 @@
 Auth = namedtuple('Auth', 'username password')
 
 
+class AskRun(namedtuple('AskRun', 'ask run')):
+    def ask_and_run(self):
+        ask_user(self.ask())
```

Comment:
ok, that's what i figured would happen but thought it was worth confirming, thanks